scripts/docker/Dockerfile

```diff
@@ -4,6 +4,10 @@ MAINTAINER Devis Lucato (https://github.com/dluc)
 
 LABEL Tags="Azure,IoT,Solutions,React,SPA"
 
+ARG user=pcsuser
+
+RUN adduser -D -h /home/$user -s /bin/bash $user $user
```

Comment:
```
RUN groupadd -r $user && useradd --no-log-init -r -g $user $user
```

Reply:
'groupadd' and 'useradd' command are not supported in /bin/sh. I will change them to use 'addgroup' and 'adduser' which works in sh.